### PR TITLE
feature_xpub turned on in testing environments

### PR DESCRIPTION
**Pull request by giorgiosironi**
Second attempt of https://github.com/elifesciences/builder-configuration/pull/8

### CODE REVIEW

File: pillar/environment-prod-public.sls [hunk 1]
```diff
@@ -13,3 +13,8 @@ elife_xpub:
     s3:
         bucket: prod-elife-xpub
     deployment_target: prod
+
+journal:
+    feature_xpub: false
+    submit_url: http://submit.elifesciences.org/
```
Comment (thewilkybarkid) [inline pillar/environment-prod-public.sls]: Better as `https`.

Comment (giorgiosironi) [reply pillar/environment-prod-public.sls]: I copied this from https://elifesciences.org, but it supports HTTPS. Was it intentional?

Comment (giorgiosironi) [reply pillar/environment-prod-public.sls]: https://github.com/elifesciences/builder-configuration/blob/master/pillar/journal-public.sls#L2 is indeed https, but has never been used.

Comment (thewilkybarkid) [reply pillar/environment-prod-public.sls]: Journal looks never to have been updated to make it HTTP (suspect it wasn't available over HTTPS at the time). Was going to make the switch now.

**Pull request merged**